setup.py

```diff
@@ -46,6 +46,33 @@
         'console_scripts': [
             'custodia = custodia.server:main'
         ],
+        'custodia.authenticators': [
+            'simplecreds = custodia.httpd.authenticators:SimpleCredsAuth',
```

Comment:
The plugin may be, but lloking at the config file now it is a litte bit less clear.
I think it woiuld help to retain the original names, camel cased, so I was asking why the change, is it just personal aestethic preference ? Or is there some more technical/style/etc reason behind it ?